custom/icds_reports/utils/aggregation_helpers/child_health_monthly.py

```diff
@@ -335,7 +343,10 @@ def pre_aggregation_queries(self):
         return [self._state_aggregation_query(state_id) for state_id in self.state_ids]
 
     def create_temporary_table(self):
-        return "CREATE TABLE \"{}\" (LIKE child_health_monthly INCLUDING INDEXES)".format(self.temporary_tablename)
+        return """
+        CREATE UNLOGGED TABLE \"{table}\" (LIKE child_health_monthly INCLUDING INDEXES);
```

Comment:
@calellowitz FYI any temporary tables that we create that aren't real temp tables should be 'unlogged' which give much better performance.